api/src/test/scala/hmda/api/protocol/admin/WriteInstitutionProtocolSpec.scala

```diff
@@ -20,19 +20,110 @@ class WriteInstitutionProtocolSpec extends PropSpec with PropertyChecks with Mus
       i.toJson mustBe
         JsObject(
           ("id", JsString(i.id)),
-          ("name", JsString(i.name)),
-          ("cra", JsBoolean(i.cra)),
           ("agency", JsString(i.agency.name)),
+          ("activityYear", JsNumber(i.activityYear)),
+          ("institutionType", JsString(i.institutionType.entryName)),
+          ("cra", JsBoolean(i.cra)),
           ("externalIds", JsArray(i.externalIds.map { x =>
             JsObject(
               ("id", JsString(x.id)),
               ("idType", JsString(x.idType.entryName))
             )
           }.toVector)),
-          ("hasParent", JsBoolean(i.hasParent)),
-          ("institutionType", JsString(i.institutionType.entryName))
+          ("emailDomains", JsArray(i.emailDomains.map { e =>
+            JsString(e)
+          }.toVector)),
+          ("respondent", JsObject(
+            ("externalId", JsObject(
+              ("id", JsString(i.respondent.externalId.id)),
+              ("idType", JsString(i.respondent.externalId.idType.entryName))
+            )),
+            ("name", JsString(i.respondent.name)),
+            ("state", JsString(i.respondent.state)),
+            ("city", JsString(i.respondent.city)),
+            ("fipsStateNumber", JsString(i.respondent.fipsStateNumber))
+          )),
+          ("hmdaFilerFlag", JsBoolean(i.hmdaFilerFlag)),
+          ("parent", JsObject(
+            ("respondentId", JsString(i.parent.respondentId)),
+            ("idRssd", JsNumber(i.parent.idRssd)),
+            ("name", JsString(i.parent.name)),
+            ("city", JsString(i.parent.city)),
+            ("state", JsString(i.parent.state))
+          )),
+          ("assets", JsNumber(i.assets)),
+          ("otherLenderCode", JsNumber(i.otherLenderCode)),
+          ("topHolder", JsObject(
+            ("idRssd", JsNumber(i.topHolder.idRssd)),
+            ("name", JsString(i.topHolder.name)),
+            ("city", JsString(i.topHolder.city)),
+            ("state", JsString(i.topHolder.state)),
+            ("country", JsString(i.topHolder.country))
+          ))
         )
     }
   }
 
 }
+/*
+{"otherLenderCode":7,
```

Comment:
Should this comment stay here? If so, it would be nice if it had a label/explanation.